test/system/upgrade_test.sh

```diff
@@ -65,13 +65,13 @@ echo "compact -t test_ingest -w" | $CURR/bin/accumulo shell -u root -p secret
 echo "compact -t test_ingest -w" | $CURR/bin/accumulo shell -u root -p secret
 "$CURR/bin/accumulo" org.apache.accumulo.test.VerifyIngest --size 50 --timestamp 2 --random 57 --rows 500000 --start 0 --cols 1 -i $INSTANCE -u root -p secret
 
-"$CURR/bin/stop-all.sh"
-"$CURR/bin/start-all.sh"
+"$CURR/bin/accumulo-cluster stop"
+"$CURR/bin/accumulo-cluster start"
 
 "$CURR/bin/accumulo" org.apache.accumulo.test.VerifyIngest --size 50 --timestamp 2 --random 57 --rows 500000 --start 0 --cols 1 -i $INSTANCE -u root -p secret
 
 pkill -9 -f accumulo.start
-"$CURR/bin/start-all.sh"
+"$CURR/bin/accumulo-cluster start"
```

Comment:
I think the quoting here is wrong. Start is the first arg. Only the first part needs to be quoted (actually, only `$CURR` needs it, but doesn't hurt to include the full path to the script).